Index: qsparql/src/sparql/kernel/qsparqlconnection.cpp
===================================================================
--- qsparql.orig/src/sparql/kernel/qsparqlconnection.cpp	2011-03-03 15:16:29.000000000 +0200
+++ qsparql/src/sparql/kernel/qsparqlconnection.cpp	2011-03-03 15:20:33.000000000 +0200
@@ -343,6 +343,14 @@
 drivers, use drivers().  The accepted connection options depend on the selected
 driver.  The drivers ignore unneeded connection options.
 
+Meego: The security tokens required to work depend on the specific driver:
+@credential TrackerReadAccess To run queries via DBus. 
+  Mandatory for queries with QTRACKER driver.
+@credential TrackerWriteAccess To run any update. 
+  Mandatory for updates with QTRACKER or QTRACKER_DIRECT drivers.
+@credential GRP::metadata-user Permission to run queries directly on the DB. 
+  Mandatory for queries with QTRACKER_DIRECT
+
 */
 QSparqlConnection::QSparqlConnection(const QString& type,
                                      const QSparqlConnectionOptions& options,
